.azure/pipelines/jobs/default-build.yml

```diff
@@ -263,12 +256,3 @@ jobs:
           testResultsFiles: '**/artifacts/log/**/*.junit.xml'
           buildConfiguration: $(BuildConfiguration)
           buildPlatform: $(AgentOsName)
-      - task: PublishTestResults@2
```

Comment:
And this?

Reply:
Because TEST-junit-jupiter.xml also does not exist even in signalr-daily-tests runs. The `junit` result files I see under artifacts/logs/ have names like win32.signalr.junit.xml. In any case, the name seems more important in Helix test runs (see [last few lines of src/SignalR/clients/java/signalr/signalr.client.java.Tests.javaproj](https://github.com/dotnet/aspnetcore/blob/3132e5c6cee173bc025f36b2cfa3c53161370ca1/src/SignalR/clients/java/signalr/signalr.client.java.Tests.javaproj#L87-L88)). It's in Helix runs that I see e.g. `com.microsoft.signalr.HubConnectionTest` tests reported.

@BrennanConroy please confirm this file won't exist on our build agents, just on Helix machines.

Reply:
The Java tests moved to Helix and should only be running there now. However, I think I still saw signalr-daily running them (which it shouldn't), so this is fine.